src/include/catalog/storage_pending_deletes_redo.h

```diff
@@ -0,0 +1,46 @@
+/*-------------------------------------------------------------------------
+ *
+ * storage.h
+ *	  prototypes for functions in backend/catalog/storage_pending_deletes_redo.c
+ *
+ * Copyright (c) 2025 Greengage Community
+ *
+ * IDENTIFICATION
+ * src/include/catalog/storage_pending_deletes_redo.h
+ *
+ *-------------------------------------------------------------------------
+ */
+#ifndef STORAGE_PENDING_DELETES_REDO_H
+#define STORAGE_PENDING_DELETES_REDO_H
+
+#include "postgres.h"
+
+#include "access/xlog.h"
+#include "nodes/pg_list.h"
+#include "storage/relfilenode.h"
+
+/* Pending delete node linked to xact it created */
+typedef struct PendingRelXactDelete
+{
+	RelFileNodePendingDelete relnode;
+	TransactionId xid;
+}	PendingRelXactDelete;
+
+typedef struct PendingRelXactDeleteArray
+{
+	size_t		count;
+	PendingRelXactDelete array[FLEXIBLE_ARRAY_MEMBER];
+}	PendingRelXactDeleteArray;
```

Comment:
What about moving PendingRelXactDeleteArray and PendingRelXactDelete to src/include/catalog/storage_pending_deletes.h?